packages/demo-gatsby/src/templates/blog-post.js

```diff
@@ -248,6 +273,18 @@ const BlogPostForm = {
       name: "frontmatter.draft",
       component: "toggle",
     },
+    {
+      label: "New Shiny Select",
+      name: "frontmatter.cool",
+      component: "select",
+      options: [100, "Love this!", "How cool!"]
+    },
+    {
+      label: "Testing Number Component",
+      name: "frontmatter.testNumber",
+      component: "number",
+      steps: 3
+    },
```

Comment:
This outputs as `Unrecognized field type` in the sidebar; maybe this shouldn't be a part of the select PR? Otherwise this is good to merge in, tested and looks good. This is meant to be a simple first step, after all.